website/docs/introduction.md

```diff
@@ -3,33 +3,69 @@ id: introduction
+
+This website for example, is build with Docusaurus 2 alpha.
 
 ## Features
 
-TODO
+- ⚛️ **Built using React** Extend or customize with React.
+- ⚡️ **Super Fast** Link prefetching, instant navigation and page views
+- ✂️ **Easy** Hot Reloadable out-of-the-box, automatic code and data splitting!
+- 💥 **Single-page application** Add dynamic interactivity to your website (CSR)
+- 🎯 **SEO Friendly** files are staticly generated (SSR)
+- 😌 **Painless** Painless project setup. First class support for documentation
+- 📝 **MDX Based.** Write JSX and React in markdown 
+- ☁️ **GitHub pages friendly** Publish to GitHub pages with one command !
+- 🔌 **Pluggable** Plugin system to extend basic functionality
+- 🎨 **Themeable** Theming system to customize how your website's appearance
+- 🔍 **Search** Make it easy for people to search what they need in your website.
+- 🌎 **i18n** Internationalize your website easily (*coming soon*) 
+- 💾 **Version** Versioning support (*coming soon*) 
+- 🚀 **many others** ....
+
+
+## Comparison with other tools
+
+### Docz
+
+Docz is a great project but it is best suited to internally document a React component library. It doesn't help you generate a nice looking landing page or blog like what Docusaurus does, you have to write your own MDX for all of that. It is also completely runtime-driven (not a static site), doesn't work without JavaScript and therefore not SEO friendly.
+
+## GitBook
+
+The primary problem with GitBook is that the team behind it is more focused on turning it into a commercial product rather than an open-source tool. It also has a lot of problems that Docusaurus doesn't have. That's why [Redux switched to Docusaurus](https://github.com/reduxjs/redux/issues/3161). It is only free for open-source and & non-profit teams. Docusaurus is free for everyone.
+
+## Gatsby
+
+Gatsby is capable of doing what Docusaurus does. But it is designed to build various kind of websites and apps. In comparison, Docusaurus is very focused on content-centric sites and provides features tailed for documentation of the box. 
+Gatsby tries to **do many things well**, while Docusaurus tries to **do one thing super well**.
+
+In addition, Gatsby's codebase is complex and it has hundreds of open issues. In comparison, Docusaurus codebase is very minimal and it has less open issues. It is much harder to fix a bug caused by Gatsby yourself unless you have a vast understanding of Gatsby's monorepository which has has ~~9000~~ over 100 packages. You'll have to rely on the maintainers who have few other hundreds of issues to solve.
+
+In addition, in order to achieve what Docusaurus does in Gatsby, you'll have to invest a lot of effort and build everything yourself. This of course give you an advantage of flexibility because you will be doing mostly everything on your own. However, that just means you're building another version of Docusaurus. Why reinvent the wheel ? 
```

Comment:
This too. Gatsby people aren't gonna be happy reading this, and this will not be true when they launch themes.

Reply:
Sure thing !!

This is roughly just what came to my mind. Definitely need an edit.
Maybe I should add **Even if there is a Gatsby Theme that achieve that, that just means someone is building "another version of Docusaurus" again**